### PR TITLE
Issue #3931: Split and Organize Checkstyle inputs by Test for NoWhitespaceAfter

### PR DESCRIPTION
**Pull request by Kietzmann**
This pull request solves the issue #3931 
Moved appropriate classes into new package. Maven clean verify completed successfully.

### CODE REVIEW

**Comment by romani** [comment]
@Kietzmann , please read errors from Travis and resolve it

**Comment by Kietzmann** [comment]
@romani 
Could you please tell me how to rerun Travis without new commits, cause it failed because there wasn't reference to issue in Pull Request Description?
Thanks

**Comment by rnveach** [comment]
@Kietzmann It is passing now.
In the future, you have to amend a commit and repush it to re-run CIs.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterFormerNpe.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespaceafter;
+
+public class InputNoWhitespaceAfterFormerNpe
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterFormerNpe.java]: This input file looks like it is only used by this test.
We expect alone files to be moved, not copied, to their new destination.
Please confirm for this file, and fix including any other files for this test.

Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterFormerNpe.java]: @rnveach Fixed, please review

Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterFormerNpe.java]: @rnveach 
Could you please review it again. 
At last push, I faced with the problem that TeamCity build and wercker pipeline are not passing. TeamCity arguing on files that I haven't modified and also I haven't access to wercker.
Thanks

**Comment by rnveach** [comment]
@Kietzmann TC can be ignored as we are trying to fix it.
We cannot restart wercker, it is an issue on their side. You will have to amend the commit with no changes and re-push it to restart wercker.

**Comment by Kietzmann** [comment]
@rnveach 
I did it before the last commit and the result was the same. I will try to do it after few hours, let see will the result will change.

**Comment by Kietzmann** [comment]
@rnveach 
Could you please review it?

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterArrayDeclarations2.java [hunk 2]
```diff
@@ -9,28 +9,28 @@
 
     public class A {
         public int[][] create(int i, int j) {
-            return new int[3] [3];//incorrect, 12:30
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterArrayDeclarations2.java]: We are expecting very few changes in inputs.
Only allowed changes are package and class name changes. No other changes.

Please revert these superfluous changes.
Please manually confirm there are no changes like this for other inputs as we can't see differences for new input files.

**Comment by Kietzmann** [comment]
@rnveach 
Seems to be fixed(IDEA was autoformat before my commit)

**Comment by Kietzmann** [comment]
@rnveach 
Please make review

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheckTest.java [hunk 3]
```diff
@@ -73,7 +75,7 @@ public void testDefault() throws Exception {
             "136:12: " + getCheckMessage(MSG_KEY, "."),
             "264:2: " + getCheckMessage(MSG_KEY, "."),
         };
-        verify(checkConfig, getPath("InputWhitespace.java"), expected);
+        verify(checkConfig, getPath("InputNoWhitespaceAfterWhitespace.java"), expected);
```
Comment (sabaka) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheckTest.java]: Name "NoWhitespaceAfterWhitespace" is confusing to me.
We don't have InputNoWhitespaceAfter.java in this folder yet. Please rename file.

**Comment by romani** [comment]
@Kietzmann , please reply to each point of review as "done" to send a signal that you are ready for next review.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheckTest.java (hunk 3 above)
Comment (Kietzmann) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheckTest.java]: @sabaka 
Done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterArrayDeclarations2.java (hunk 2 above)
Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterArrayDeclarations2.java]: @rnveach 
Done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterFormerNpe.java (hunk 1 above)
Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterFormerNpe.java]: @rnveach 
Done

**Pull request merged**